lua/gitignore/init.lua

```diff
@@ -96,7 +80,19 @@ local function createGitignore(selectionList, order)
     return x
 end
 
-local function createGitignoreBuffer(chosen_path, ignoreLines, prompt_bufnr)
+--- Creates the gitignore to new buffer or new file if path is provided.
+---@param chosen_path? string path to .gitignore
+---@param selectionList table list of selected templateNames
+---@param prompt_bufnr? number bufnr for closing telescope plugin if exists
+function M.createGitignoreBuffer(chosen_path, selectionList, prompt_bufnr)
+    local ignoreLines = createGitignore(selectionList, order_data)
+    if #ignoreLines < 1 then
+        return
+    else
```

Comment:
> if there are more than 0 ignore lines, close the telescope prompt and print an error.

I am not sure if I can follow the logic for this else block, as it will throw print an error message when successfully generating a .gitignore buffer.

I believe this is a bug, and if so please fix it here and all other possible occurrences of this logic, if there are any.
Should this however be intended behavior, please elaborate on the intended functionality.

Reply:
Ahh, my bad. It really is my mistake.
After reading some code I found one interesting thing. The minimum size of the `ignoreLine` table is two elements (if you pass an empty table to the method) if you don't want to change the `generateGitignore` method.
I'll fix it.